begin.css

```diff
@@ -0,0 +1,482 @@
+/* General */
```

Comment:
- [ ] Elements should be aligned with flex, grid, and float. For this project you only use flex and float. Please align items with grid. 
> ![image](https://user-images.githubusercontent.com/57414789/82390783-3ef55180-9a05-11ea-97f3-c4d5b9862e70.png)

Reply:
This requirement was done as asked.